comps/comps-foreman-rhel7.xml

```diff
@@ -480,8 +480,8 @@
       <packagereq type="default">tfm-rubygem-hirb-unicode-steakknife-doc</packagereq>
       <packagereq type="default">tfm-rubygem-hocon-doc</packagereq>
       <packagereq type="default">tfm-rubygem-hoe-doc</packagereq>
-      <packagereq type="default">tfm-rubygem-httpclient-doc</packagereq>
```

Comment:
I think none of the changes in this file is actually relevant, mind reverting it?